src/js/components/atoms/label-filter/label-filter.scss

```diff
@@ -1,41 +1,45 @@
 /** @format */
 
-@import "~scss/settings/_variables.scss";
+@import "~scss/settings/variables";
+@import "~scss/tools/mixins/helpers";
 
-// http://rikschennink.nl/thoughts/implicit-content-and-screen-readers/
 .label-filter input[type="checkbox"] {
-    position: absolute;
-    overflow: hidden;
-    height: 1px;
-    width: 1px;
-    padding: 0;
-    border: 0;
-    clip: rect(1px, 1px, 1px, 1px);
-    clip-path: inset(50%);
-    white-space: nowrap;
+    @include visually-hidden;
 }
 
-.label-filter input[type="checkbox"]:checked + .label-filter__label {
-    color: $colorWhite;
+.label-filter__label {
+    background-color: $colorWhite;
+    border-radius: 2px;
+    border: 1px solid $colorGrayDark;
+    color: $colorBlack;
+    cursor: pointer;
+    display: inline-block;
+    font-size: 12px;
+    font-weight: 500;
+    padding: 0 $buildingUnit / 2;
+}
+
+.label-filter > :checked:not(:disabled) + .label-filter__label {
```

Comment:
I don't think `:not()f` is needed here.

Reply:
want it to be specific so there's no need for css overwrite